mediawiki/LocalSettings.d/LocalSettings.override.php

```diff
@@ -76,9 +78,9 @@
 wfLoadExtension( 'TwitterWidget' );
 
 $wgWBRepoSettings['formatterUrlProperty']='P10';
-$wgMathDisableTexFilter = 'always';
+# $wgMathDisableTexFilter = 'always';
 
-$wgMathLaTeXMLUrl = 'http://latexml:8080/convert/';
+# $wgMathLaTeXMLUrl = 'http://latexml:8080/convert/';
```

Comment:
I am pretty sure that it did work with LaTeXML rendering. The problem is probably the network=host option